### ISSUE TITLE
projectManager.saveProject not dound

### ISSUE THREAD

**Issue by ceckenrode** [alefragnani/vscode-project-manager#80]
<img width="723" alt="screen shot 2017-01-21 at 2 22 44 pm" src="https://cloud.githubusercontent.com/assets/9385814/22177074/1cc255d2-dfe5-11e6-918e-7f38450d7911.png">

I get this whenever I try to save a project, and get similar messages when trying to edit or view projects.

<img width="754" alt="screen shot 2017-01-21 at 2 20 53 pm" src="https://cloud.githubusercontent.com/assets/9385814/22177081/32bc6fd0-dfe5-11e6-9627-4f96e264e03e.png">

This started when I needed to edit the projects.json, I tried to removed a project from there, and I guess the json became malformed.

I get these errors as VSCode starts

<img width="802" alt="screen shot 2017-01-21 at 2 26 08 pm" src="https://cloud.githubusercontent.com/assets/9385814/22177092/9362230c-dfe5-11e6-9cc6-73322a334413.png">

And then these when I try and use the extension

<img width="798" alt="screen shot 2017-01-21 at 2 26 56 pm" src="https://cloud.githubusercontent.com/assets/9385814/22177097/bb13bdde-dfe5-11e6-95c6-9a30c3d82c77.png">

I've tried uninstalling and reinstalling the extension, as well as clearing out the extensions folder in the .vscode directory. I still get the same error.

Any ideas? Seems strange that uninstalling the extension doesn't resolve this when it's reinstalled. I'm running Mac OSX 10.11. Didn't have this issue before altering projects.json when it came up when I needed to edit a project

**Comment by alefragnani** [comment]
Hi @ceckenrode,

Could you paste your `projects.json` file here, to check if it is malformed? If not, you could _rename it_ so the extension should start empty and you will be sure the problem is related to this.

It is not located in extensions folder. That's why didn't make any difference when you reinstalled the extension.

Hope this helps

**Comment by ceckenrode** [comment]
Seems to be resolved now. I was a little unclear about **where** the projects.json was supposed to be. I had previously been using this extension without ever having to specify a path for that or ever having to deal with a projects.json file. Although I haven't had to 'edit' projects either, I've only added and opened them up until now.

I created a new projects.json file containing an empty array, saved this to another location on my computer, and specified the path inside of user settings under the "projectManager.projectsLocation" option.

The next time I opened VSCode and tried to use the extension I was prompted to save a new project. Everything seems to be working now. Thanks for your help @alefragnani . Not so sure this is actually a 'bug' so much as me not being able to figure out where projects.json was supposed to be location. 

Going to mark this closed, hopefully it can help if someone else runs into something similar.
